dockstore-webservice/src/main/java/io/dockstore/webservice/core/webhook/GitCommit.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.dockstore.webservice.core.webhook;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema(description = "Information about the head commit on a GitHub push event")
```

Comment:
This `GitCommit` class is used for both head commit and the list of commits so I think the schema description should be more generic (remove "head" from "head commit")